cfbs/cfbs_config.py

```diff
@@ -225,6 +227,79 @@ def _find_dependencies(self, modules, exclude):
             dependencies += self._find_dependencies(dependencies, exclude)
         return dependencies
 
+    def _add_to_inputs(self, module):
```

Comment:
These are not inputs, but build steps.

Reply:
See https://github.com/cfengine/cfbs/pull/134